trin-core/src/cli.rs

```diff
@@ -494,59 +525,134 @@ mod test {
 
     #[test]
     fn test_default_trusted_provider_is_infura() {
-        assert!(env_is_set());
         let config = TrinConfig::new_from(["trin"].iter()).unwrap();
+        assert!(env_is_set(&config));
         assert_eq!(config.trusted_provider, TrustedProviderType::Infura);
     }
 
     #[test]
-    fn test_geth_trusted_provider() {
-        assert!(env_is_set());
+    fn test_pandaops_trusted_provider() {
         let config = TrinConfig::new_from(
             [
                 "trin",
                 "--trusted-provider",
-                "geth",
-                "--geth-url",
+                "pandaops",
+                "--pandaops-url",
                 "www.geth.com",
             ]
             .iter(),
         )
         .unwrap();
-        assert_eq!(config.trusted_provider, TrustedProviderType::Geth);
+        assert!(env_is_set(&config));
+        assert_eq!(config.trusted_provider, TrustedProviderType::Pandaops);
+    }
+
+    #[test]
+    fn test_local_node_trusted_provider() {
+        use crate::utils::provider::TrustedProvider;
```

Comment:
I would move this up to the other test import statements around line `265`

Reply:
Done